developer_manual/client_apis/WebDAV/basic.rst

```diff
@@ -209,3 +328,51 @@ File properties can be requested by adding a :code:`<d:prop/>` element to the re
+^^^^^^^^^^^^^^^
+
+You can set some special headers that Nextcloud will interpret.
+
++-----------------+-----------------------------------------------------------------+----------------------------------+
+|     Header      |                           Description                           |             Example              |
++=================+=================================================================+==================================+
+| X-OC-MTime      | | Allow to specify a mtime.                                     | ``1675789581``                   |
+|                 | | The response will contain the header ``X-OC-MTime: accepted`` |                                  |
+|                 | | if the mtime was accepted.                                    |                                  |
++-----------------+-----------------------------------------------------------------+----------------------------------+
+| X-OC-CTime      | | Allow to specify a creation time.                             | ``1675789581``                   |
+|                 | | The response will contain the header ``X-OC-CTime: accepted`` |                                  |
+|                 | | if the mtime was accepted.                                    |                                  |
++-----------------+-----------------------------------------------------------------+----------------------------------+
+| OC-Checksum     | Allow to specify a checksum.                                    |                                  |
++-----------------+-----------------------------------------------------------------+----------------------------------+
+| X-Hash          | | Allow to request the file's hash from the server.             | ``md5``, ``sha1``, or ``sha256`` |
+|                 | | The server will return the hash in a header named either:     |                                  |
+|                 | | ``X-Hash-MD5``, ``X-Hash-SHA1``, or ``X-Hash-SHA256``.        |                                  |
++-----------------+-----------------------------------------------------------------+----------------------------------+
+| OC-Chunked      | Specify that the sent data is part of a chunk upload.           | ``true``                         |
++-----------------+-----------------------------------------------------------------+----------------------------------+
+| OC-Total-Length | | Contains the total size of the file during a chunk upload.    | ``4052412``                      |
+|                 | | This allow the server to abort faster if the remaining        |                                  |
+|                 | | user's quota is not enough.                                   |                                  |
++-----------------+-----------------------------------------------------------------+----------------------------------+
+
+Response Headers
+----------------
+
+Nextcloud will sometime answer with the following headers:
```

Comment:
That sounds like it depends on some random thing. Maybe just leave it out, since it's explained in the table anyway?